src/main/java/io/mosip/digitalcard/service/impl/PDFCardServiceImpl.java

```diff
@@ -159,6 +162,9 @@ public byte[] generateCard(org.json.JSONObject decryptedCredentialJson, String c
 				attributes.put("isPhotoSet",isPhotoSet);
 			}
 			uin = decryptedCredentialJson.getString("UIN");
+			if(additionalAttributes.containsKey("template")) {
+				template = additionalAttributes.get("template").toString();
+			}
```

Comment:
I still see additionalAttributes.get("cardFormat").equals("vidCard") in the code, looks like we are uisng both cardFormat and template. I think we should support only one type. Lets get it as cardType and remove others.

Reply:
taken care`